### PR TITLE
Mark this repository as obsolete

### PR DESCRIPTION
**Pull request by natemcmaster**
This repo is no longer being used to build the dev branch of aspnetcore as of https://github.com/aspnet/Universe/pull/566

Part of https://github.com/aspnet/Coherence-Signed/issues/648.

### CODE REVIEW

File: packages/packages.csv [hunk 1]
```diff
@@ -1,229 +1,2 @@
...
-Microsoft.VisualStudio.Web.CodeGeneration.Design,ship,exclude
-Microsoft.VisualStudio.Web.CodeGeneration.EntityFrameworkCore,ship,exclude
-Microsoft.VisualStudio.Web.CodeGeneration.Templating,ship,exclude
-Microsoft.VisualStudio.Web.CodeGeneration.Tools,ship,exclude
-Microsoft.VisualStudio.Web.CodeGeneration.Utils,ship,exclude
-Microsoft.VisualStudio.Web.CodeGenerators.Mvc,ship,exclude
-Microsoft.Web.Xdt.Extensions,shipoob,exclude
-RazorPageGenerator,noship,exclude
+This file is no longer in use.
+Please use https://github.com/aspnet/Universe/blob/dev/build/artifacts.props instead.
```
Comment (muratg) [inline packages/packages.csv:2]: :+1:

File: README.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
...
+Coherence (obsolete)
+====================
 
 The coherence build verifies that all the packages form a consistent graph.
 
 This project is part of ASP.NET Core. You can find samples, documentation and getting started instructions for ASP.NET Core at the [Home](https://github.com/aspnet/home) repo.
+
+### Obsolete notice
+
+This repository is no longer under active development. Its functionality has moved to https://github.com/aspnet/Universe
```
Comment (muratg) [inline README.md]: little nit: End with period?

**Review by muratg — APPROVED** [review]

File: .github/ISSUE_TEMPLATE [hunk 3]
```diff
@@ -0,0 +1 @@
+This repo is obsolete. Please https://github.com/aspnet/Universe instead.
```
Comment (JunTaoLuo) [inline .github/ISSUE_TEMPLATE]: please use?

**Review by JunTaoLuo — APPROVED** [review]

File: .github/ISSUE_TEMPLATE (hunk 3 above)
Comment (natemcmaster) [reply .github/ISSUE_TEMPLATE]: Why bother good grammar if bad do fine?

**Pull request merged**